oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+partner-contact
```

Comment:
Why that's needed?

Reply:
Module depends on `partner_firstname`

Reply:
Totally makes sense, sorry, missed that :D